stdlib/2/os2emxpath.pyi

```diff
@@ -90,31 +81,31 @@ else:
         def realpath(filename: AnyStr) -> AnyStr: ...
 
 if sys.version_info >= (3, 6):
-    # In reality it returns str for sequences of _StrPath and bytes for sequences
-    # of _BytesPath, but mypy does not accept such a signature.
-    def commonpath(paths: Sequence[_PathType]) -> Any: ...
+    # In reality it returns str for sequences of StrPath and bytes for sequences
+    # of BytesPath, but mypy does not accept such a signature.
```

Comment:
Feels like this should be fixable with overloads, but that's better kept for a separate PR.